Jenkinsfile

```diff
@@ -278,7 +278,12 @@ try {
 						if (hasRegressionRunSucceeded) {
 							println 'Regression suite passed, proceeding to next stage...'
 						} else {
-							error('Regression suite failed, check the CloudWatch logs above for more details')
+							try {
+								input 'Regression suite failed, check the CloudWatch logs above for more details. Should deployment proceed?'
+								echo "Regression suite failure in '${bfdEnv}' has been accepted by operator. Proceeding to next stage..."
+							} catch(err) {
+								error "Operator opted to fail deployment due to regression suite failure in '${bfdEnv}'"
+							}
```

Comment:
@lsmitchell @christrotteradhoc @mjburling FYI for those in the Surge Status meeting and for Burling -- I've updated the handling for regression suite failures in the deployment pipeline to not immediately fail. Instead, we wait for an operator to explicitly abort or proceed via the [`input`](https://www.jenkins.io/doc/pipeline/steps/pipeline-input-step/) step, similarly to the "manual approval" stage. This allows us to avoid false negatives automatically causing deployment failures.

Let me know if any of you think this is insufficient. Also, (and this can be done in the future), should we be sending a notification when this happens so that there's better awareness? I know we monitor deployments fairly closely right now, so I don't think it's necessary at the moment but it's something I was thinking about.

Reply:
Also, this block is duplicated across the two other stages (for `prod-sbx` and `prod`) -- not just for `test`.

Reply:
This is exactly the sort of functionality that I had in mind! Is that all it takes.

Reply:
Seems so. Thanks for putting that together!